eq-author/src/App/page/Design/QuestionPageEditor/AnswersEditor/index.js

```diff
@@ -23,6 +23,10 @@ export const AnswersEditor = ({
   onDeleteAnswer,
   moveAnswer,
 }) => {
+
+  let multipleAnswers = false;
+  multipleAnswers = answers?.length > 1;
```

Comment:
```suggestion
const multipleAnswers = answers?.length > 1;
```

🎬 Take 3 ;-)

Reply:
If I do this @tbaggaley then multipleAnswers doesn't exist as a prop that gets passed down.
I guess I could make that prop not required, but seems like it really should be required

Reply:
Yeah it doesn't affect any of your other code. You're just declaring & defining `multipleAnswers` and then sticking that straight away in a prop without any further processing

Reply:
const multipleAnswers = answers?.length > 1 ? true : false;
This would make sense but the linter gets sad ☹️